docs/architecture/serverless/index.md

```diff
@@ -38,17 +38,17 @@ All other marks and logos are property of their respective owners.
 
 Author:
 
-> **[Jeremy Likness](https://twitter.com/jeremylikness)**, Sr. Cloud Developer Advocate, APEX, Microsoft Corp.
+> **[Jeremy Likness](https://twitter.com/jeremylikness)**, Senior Cloud Advocate, Microsoft Corp.
 
 Contributor:
 
-> **[Cecil Phillip](https://twitter.com/cecilphillip)**, Cloud Developer Advocate II, APEX, Microsoft Corp.
+> **[Cecil Phillip](https://twitter.com/cecilphillip)**, Senior Cloud Advocate, Microsoft Corp.
```

Comment:
I'm not sure of his current position, but the old text didn't mention `Senior`.

Reply:
I've updated according to the current title 😄